src/db/db_test.rs

```diff
@@ -1,17 +1,30 @@
 #[cfg(test)]
 mod test {
 
+    use tempfile::tempdir;
+
+    // Helper function to create tempoary directory for new database creation.
+    // This returns the path for newly created directory.
+    fn create_temp_dir() -> String {
```

Comment:
@brianshih1 Added a helper function for temporary directory creation.

Reply:
thanks, looks good!!